### PR TITLE
Challenge - add FE task

### PR DESCRIPTION
**Pull request by rafalbromirski**

### CODE REVIEW

File: frontend-engineer-challenge.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+##### Liquidity
+
+It should render a *scatter plot chart* (with received data) where:
+
+* `X` axis is Market Capitalization
+* `Y` axis is Volume (24h)
+* `Z` axis (or size of the point) is absolute price change (24h)
+
+Each point represents one coin.
```
Comment (grenzbotin) [inline frontend-engineer-challenge.md]: Maybe adding a hover effect as requirement? Scatter plots without showing at least the coin / name when hovering are kinda useless, or is this something that you would expect from them without saying to check whether they think about what they do? 🤔

Comment (rafalbromirski) [reply frontend-engineer-challenge.md]: indeed 👍

File: frontend-engineer-challenge.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+##### User interface
+
+By default `/ticker` endpoint returns top 100 coins. There should be an option to change it by using a select/dropdown with predefined values:
+
+* `10`
+* `50`
+* `100`
+* `all`
+
+Keep in mind it should have global scope and it should affect data on both pages: Market Overview and Liquidity.
```
Comment (marcinb) [inline frontend-engineer-challenge.md]: keep in mind **that** it should have...

File: frontend-engineer-challenge.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
+# Mint Front-End Engineer Challenge
+
+### Overview
+
+The service should fetch an API `/ticker` endpoint from [coinmarketcap.com)](https://coinmarketcap.com/api/) and contains
```
Comment (marcinb) [inline frontend-engineer-challenge.md]: coinmarketcap.com) -> coinmarketcap.com

File: frontend-engineer-challenge.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+a navigation with links to 2 pages:
+
+* Market overview (default page, might be `/` url)
+* Liquidity (`/liquidity` url)
+
+##### Market Overview
+
+It should render a *table* (with received data) sorted by rank and columns like:
```
Comment (marcinb) [inline frontend-engineer-challenge.md]: you might find https://ozh.github.io/ascii-tables/ helpful

File: frontend-engineer-challenge.md (hunk 2 above)
Comment (marcinb) [reply frontend-engineer-challenge.md]: Shall we add an `auto-refresh` option here as well?

File: frontend-engineer-challenge.md (hunk 3 above)
Comment (marcinb) [reply frontend-engineer-challenge.md]:
```
The service should fetch https://coinmarketcap.com/api/ API (ticker endpoint) and use the returned data to render:

* Market overview (default page, might be `/` url)
* Liquidity (`/liquidity` url)
```
wdyt?

File: frontend-engineer-challenge.md (hunk 2 above)
Comment (rafalbromirski) [reply frontend-engineer-challenge.md]: there is nothing to refresh... unless you will wait 5min ;D

File: frontend-engineer-challenge.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
+# Mint: Front-End Engineer Challenge
+
+### Overview
+
+Create a webpage that fetches data from the [coinmarketcap.com](https://coinmarketcap.com/api/) `/ticker` API endpoint.
+The page should display navigation to market overview (described below) and liquidity (described below):
```
Comment (marcinb) [inline frontend-engineer-challenge.md]: How about sth like:

>> Create webapp that allows for simple analysis of crypto assets. The page should present a navigation bar that takes you either to the Marker Overview page (`/`) or to the Liquidity analysis page (`/liquidity`).
The app should fetch all the required data from the [coinmarketcap.com](https://coinmarketcap.com/api/) `/ticker` API endpoint.

**Pull request merged**